docs/content/paket-init-auto-restore.md

```diff
@@ -0,0 +1,11 @@
+# paket init-auto-restore
```

Comment:
This file is generated from https://github.com/fsprojects/Paket/blob/master/src/Paket/HelpTexts.fs please don't commit it.